### DISCUSSION TITLE
Any way to get a rich renderable to string?

### DISCUSSION THREAD

**Discussion by confidentialsecrets** [Textualize/rich#2374] [Q&A] [answered]
I want to write Panel and/or Tree to a text file, (without color obviously) this can be done manually by copying and pasting it from the terminal after printing it but I need to automate it, any way to get the string from the Tree or Panel?

**Comment by willmcgugan** [comment, marked as answer]
https://rich.readthedocs.io/en/latest/console.html#capturing-output
